sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -765,21 +807,32 @@
      * @return the response.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Flux<ByteBuffer> getMapStaticImage(
-            RasterTileFormat format,
-            StaticMapLayer layer,
-            MapImageStyle style,
-            Integer zoom,
-            List<Double> center,
-            List<Double> boundingBox,
-            Integer height,
-            Integer width,
-            String language,
-            LocalizedMapView localizedMapView,
-            List<String> pins,
-            List<String> path) {
-        return this.serviceClient.getMapStaticImageAsync(
-                format, layer, style, zoom, center, boundingBox, height, width, language, localizedMapView, pins, path);
+    public Mono<StreamResponse> getMapStaticImageWithResponse(MapStaticImageOptions options) {
+        return this.getMapStaticImageWithResponse(options, null);
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    Mono<StreamResponse> getMapStaticImageWithResponse(MapStaticImageOptions options, Context context) {
+        BoundingBox boundingBox = options.getBoundingBox();
+        LatLong southWest = boundingBox.getSouthWest();
+        LatLong northEast = boundingBox.getNorthEast();
+        Center center = options.getCenter();
+        List<Double> centerPrivate = null;
+        if (center != null) centerPrivate = Utility.toCenterPrivate(center); //minLon, minLat, maxLon, maxLat'
+        List<Double> bbox = Arrays.asList(southWest.getLatitude(), northEast.getLongitude(), northEast.getLatitude(), southWest.getLongitude());
```

Comment:
second time you use this construct. Shouldn't it be in Utility.

List<Double> boundsasList(BoundingBox box) {
   ///
   return list
}